app/javascript/src/components/Dashboard/Contacts/constants.js

```diff
@@ -4,15 +4,15 @@ import * as yup from "yup";
 import { SINGULAR } from "constants";
 
 export const ROW_DATA = Array.from({ length: 12 }, (_, index) => {
-  const isEven = index % 2 === 0;
+  const isOdd = index % 2;
```

Comment:
The value should be converted to a boolean value for explicitness if you are going with this approach. I preferred the conditional check one that you had before.